src/applications/vaos/appointment-list/index.jsx

```diff
@@ -20,7 +20,10 @@ function AppointmentListSection() {
     <Switch>
       <Route path="/cc/:id" component={CommunityCareAppointmentDetailsPage} />
       <Route path="/va/:id" component={ConfirmedAppointmentDetailsPage} />
-      <Route path="/requests/:id" component={RequestedAppointmentDetailsPage} />
+      <Route
+        path="/requests/:id/:confirmMsg?"
```

Comment:
Can we make confirmMsg a query parameter, instead of part of the path? I think that's clearer